pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -96,6 +96,7 @@ stdenv.mkDerivation {
   name = targetPrefix
     + (if name != "" then name else stdenv.lib.removePrefix targetPrefix "${ccName}-wrapper")
     + (stdenv.lib.optionalString (cc != null && ccVersion != "") "-${ccVersion}");
+  version = ccVersion;
```

Comment:
My https://github.com/NixOS/nixpkgs/pull/67284#discussion_r349885907 was never addressed. If we are really going to break the case when `cc == null`, we might as well change this to provide pname.

Reply:
Ah never mind, didn't notice 591034b45a83f28734afdc5ed787141f8494a6cc . This is not very pretty.

Reply:
Oh, sorry I missed that part. I agree.